include/dsn/cpp/utils.h

```diff
@@ -425,7 +424,7 @@ namespace dsn {
 
         typedef struct _tls_tid
         {
-            int magic;
+            unsigned int magic;
```

Comment:
I think this is great in that case, and I don't realize that 0xdeadbeef is unsigned:)